templates-blocks/background-section/background-section.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * UDS Background Section
+ *
+ * @package UDS WordPress Theme
+ */
+
+$choice = get_field( 'uds_background_section_choice' );
+$pattern = get_field( 'uds_background_section_pattern' );
+$preset = get_field( 'uds_background_section_preset' );
+$color = get_field( 'uds_background_section_color' );
+$upload = get_field( 'uds_background_section_upload_url' );
+$innercolor = get_field( 'uds_background_inner_color' );
+$repeat = get_field( 'uds_background_image_repeat' );
+$position = get_field( 'uds_background_image_position' );
+
+// Retrieve additional classes from the 'advanced' field in the editor.
+if ( ! empty( $block['className'] ) ) {
+	$additional_classes = $block['className'];
+}
+
```

Comment:
Can you give the $additional_classes an empty value if it's empty to avoid the following error: 
`Notice: Undefined variable: additional_classes`

Reply:
Resolved. Good catch. :-)